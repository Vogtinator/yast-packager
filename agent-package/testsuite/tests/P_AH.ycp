{
    return  Execute(.setEnvironment, $["packageinfopath":"tests","language":"german","common.pkd":"baum.pkd","dudir":"tests/du.dir","partition":[$["name":"/","used":0,"free":10000],$["name":"bin","used":0,"free":10000],$["name":"etc","used":0,"free":10000],$["name":"sbin","used":0,"free":10000],$["name":"usr","used":0,"free":10000],$["name":"var","used":0,"free":10000]]]);
    }
{
    return Read(.isInstallSelected);
}
{
    return Execute(.setInstallSelection,["a"], false);
}
{
    return Read(.isInstallSelected);
}
{
    return Read(.dependencies);
}
