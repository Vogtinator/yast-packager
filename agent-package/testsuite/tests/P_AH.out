(true)
(false)
(true)
(true)
($["ADD":["b", "e"], "CONFLICT":[], "OBSOLETE":[], "REQUIRE":[]])
